framework/core/allocated.h

```diff
@@ -260,7 +260,7 @@ class AllocatedBase
   protected:
 	virtual void           post_create(VmaAllocationInfo const &allocation_info);
 	[[nodiscard]] VkBuffer create_buffer(VkBufferCreateInfo const &create_info);
-	[[nodiscard]] VkImage  create_image(VkImageCreateInfo const &create_info);
+	[[nodiscard]] VkImage  create_image(VkImageCreateInfo &create_info, VkImageCompressionControlEXT const &compression_control = {});
```

Comment:
Per my last comment, I think this function shouldn't need to be modified, because by the time we get here, the `pNext` chain for `create_info` should already be populated.

Reply:
Addressed in my last commit